File: backend/src/main/java/cohappy/backend/model/dto/request/RegisterDTO.java
package cohappy.backend.model.dto.request;

import lombok.Data;

import java.util.List;

@Data
public class RegisterDTO {
    private String name;
    private String surname;
    private String birthDate;
    private String cf;
    private List<byte[]> images;
    private String email;
    private String phoneNumber;
    private String password;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/RegisterDTO.kt
package cohappy.frontend.model.dto.request

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class RegisterDTO(
    val name: String,
    val surname: String,
    val birthDate: String,
    val cf: String,
    var images: List<ByteArray>? = null,
    val email: String,
    val phoneNumber: String,
    val password: String
)
